workflow/scripts/update-sample-sheet.py

```diff
@@ -107,6 +107,23 @@ def update_sample_sheet(SAMPLE_SHEET, verbose=True, dry_run=False):
             print("\tIn total: {}".format(i))
 
     files_to_copy = [f for f in incoming_files if f not in data_files]
+
+    # get flag showing inclusion in rki-report
+    sample_list = []
+    include_data_list = []
+    for f in files_to_copy:
+        name_sample = f.split("_")[0]
+        if name_sample not in sample_list:
+            sample_list.append(name_sample)
+            if "no-rki" in name_sample.casefold():
+                include_data = str(0)
+            else:
+                include_data = str(1)
+            include_data_list.append(include_data)
+    include_in_data_df = pd.DataFrame(
+        {"include_in_high_genome_summary": include_data_list}, index=sample_list
+    )
```

Comment:
It would be easier, more efficient and "robust" to use Padas filtering options on the finished sample name in the `new_files_df`. See here for more information: https://towardsdatascience.com/check-for-a-substring-in-a-pandas-dataframe-column-4b949f64852. (Use your browsers incognito mode, if they want to create an account)